### DISCUSSION TITLE
Is there anyway to run terminal as an independent process from code-server?

### DISCUSSION THREAD

**Discussion by ky1emoon** [coder/code-server#5590] [Q&A] [answered]
When I'm running a cpu-heavy job and closing the browser (which leads to disconnection log from code-server),
the terminal running the CPU-heavy job under code-server process goes to a sleeping state with the code-server process (S state in top).

I wonder if there are some other ways to run the terminal as an independent process using a code server client. 

Thanks in advance!

**Comment by jsjoeio** [comment]
I don't think so because it's part of VS Code 🤔 @code-asher can confirm though

**Reply by code-asher** [reply]
Yup, it uses VS Code's communication channels so it is inseparable as things currently stand.

**Comment by ky1emoon** [comment, marked as answer]
Just launched screen process along with the code-server process (`screen -dmS permanent /bin/bash && code-server --bind-addr --some-option`), and accessing screen process from code-server (`screen -r permanent`) provides permanent process terminal.

**Reply by jsjoeio** [reply]
Amazing!

**Reply by P1YU5H-50N1** [reply]
Thanks a ton!
Any way I can make it work with the docker container?
